ProjFS.Mac/PrjFSLib/PrjFSLib.cpp

```diff
@@ -1505,24 +1505,40 @@ static const char* GetRelativePath(const char* fullPath, const char* root)
         return "";
     }
     
+    string altPath(fullPath);
+    string altRoot(root);
+    const char* removePart = "/System/Volumes/Data";
+    const char* realFullPath;
+    if (altPath.find(removePart) == 0 && altRoot.find(removePart) != 0)
+    {
+        // If fullPath contains "/System/Volumes/Data", but the root does not remote it for comparison
+        // "/System/Volumes/Data/Users/account" is effectively "/Users/account" and should be compared as such
```

Comment:
If you clone 'via' `System/Volumes/Data/Users/account/Repos/MyRepo`, you'll fail events because they don't come in with the `System/Volumes/Data` prefix like you suggest.  I tested this out to verify.

I'm wondering if this is something we want to try to support or can we just ask users to go via /Users/account.  
It may be as simple as stripping the reverse, but I'm worried we're opening a can or worms potentially.

Reply:
> I'm wondering if this is something we want to try to support or can we just ask users to go via /Users/account.
It may be as simple as stripping the reverse, but I'm worried we're opening a can or worms potentially.

I think we should support this, but I don't think it needs to hold up this PR.  It sounds like it's potentially related to #1358, I will update that work item.

Reply:
I'll mark this comment as resolved

Reply:
>  && (strlen(root) < removePartLength || (strncmp(root, removePart, removePartLength) != 0))

As discussed offline, rather than trying to handle `root` starting with `removePart` here, it would be cleaner to strip it off at clone/mount time.

As it is now if the root starts with `System/Volumes/Data` none of the other notifications will work correctly.